Tests/eppo/ClientTests.swift

```diff
@@ -172,7 +172,17 @@ final class eppoClientTests: XCTestCase {
                     XCTFail("Unknown value type: \(testCase.valueType)");
             }
             
+
             XCTAssertTrue(loggerSpy.wasCalled, "Assignment logger was not called.")
+            if let lastAssignment = loggerSpy.lastAssignment {
+                XCTAssertTrue(lastAssignment.experiment.contains(lastAssignment.allocation))
```

Comment:
I'd expect an equality check for an exact match. I think the test case's "experiment" is really the flag key, so we'd want the last assignments experiment to equal `testCase.experiment + '-' + lastAssignment.allocation`.

A contains check would pass even with the bug you fixed I think.